src/onboarding/welcome/Welcome.tsx

```diff
@@ -28,6 +28,8 @@ export default function Welcome() {
 
   const createAccountCopyTestType = useSelector(createAccountCopyTestTypeSelector)
 
+  const guidedOnboardingEnabled = useSelector(showGuidedOnboardingSelector)
```

Comment:
nit: I'd appreciate if these were consistent wrt naming. I.e. use the variable `showGuidedOnboarding` for the `showGuidedOnboardingSelector`